### PR TITLE
Merge variable declaration and assignment.

### PR DESCRIPTION
**Pull request by rayandas**
Declaring and assigning variables can be done together, simplifying the code.

### CODE REVIEW

File: prow/config/config.go [hunk 1]
```diff
@@ -1097,26 +1097,23 @@ func yamlToConfig(path string, nc interface{}) error {
 		jc = &v.JobConfig
 	}
 	for rep := range jc.PresubmitsStatic {
-		var fix func(*Presubmit)
-		fix = func(job *Presubmit) {
+		var fix func(*Presubmit) = func(job *Presubmit) {
```
Comment (fejta) [inline prow/config/config.go:1100]: Won't this work?

```golang

fix := func(job *Presubmit) {
 ///
}
```

**Review by fejta — COMMENTED** [review]
/ok-to-test

File: prow/config/config.go (hunk 1 above)
Comment (rayandas) [reply prow/config/config.go]: @fejta Ah! Right.

**Comment by rayandas** [comment]
@fejta I am really sorry. I messed up. I will raise a fresh PR. Closing this.

**Pull request closed without merging**